outport/process/transactionsfee/transactionsFeeProcessor.go

```diff
@@ -249,6 +271,19 @@ func (tep *transactionsFeeProcessor) prepareTxWithResultsBasedOnLogs(
 
 	for _, event := range txWithResults.log.GetLogEvents() {
 		if core.WriteLogIdentifier == string(event.GetIdentifier()) && !hasRefund {
+			if !check.IfNilReflect(userTx) && tep.enableEpochsHandler.IsFlagEnabledInEpoch(common.FixRelayedBaseCostFlag, epoch) {
```

Comment:
do you need check.IfNilReflect ? don't you have simple check.IfNil ?

Reply:
updated